client/src/legacy/GridField.js

```diff
@@ -12,6 +12,7 @@ import '../../../thirdparty/jquery-entwine/jquery.entwine.js';
 $.entwine('ss', function($) {
   $('.grid-field').entwine({
     onmatch: function () {
+      // Adds an actions column for the search action if it does not exist already
```

Comment:
Please remove this comment from here and add it where this actually happens - probably on the line just before `    fixColumns: function (visible) {` would be perfect.